src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php

```diff
@@ -36,10 +67,10 @@ class TemplateController implements ContainerAwareInterface
      */
     public function templateAction($template, $maxAge = null, $sharedAge = null, $private = null)
     {
-        if ($this->container->has('templating')) {
-            $response = $this->container->get('templating')->renderResponse($template);
-        } elseif ($this->container->has('twig')) {
-            $response = new Response($this->container->get('twig')->render($template));
+        if ($this->templating) {
+            $response = $this->templating->renderResponse($template);
```

Comment:
the interface of the component, that you used in the typehint does not have the `renderResponse` method

Reply:
good catch, fixed, thanks

Reply:
I would have done the opposite. Not relying on the interface from frameworkbundle and create the response by hand as done for Twig.